wikimedia/wikipedia-ios
# Image recommendations - Present gallery after tapping on image

**Phabricator:** https://phabricator.wikimedia.org/T358910

### Notes
This PR leans on the existing `MediaListGalleryViewController` to present the gallery view from Image Recommendations.

### Test Steps
1. On Staging, go to Image Recommendations
2. Tap image (just a gray box for now)
3. Confirm gallery launches. Dismiss gallery
4. Tap Not sure
5. Tap image again, confirm gallery launches with new image.
